drivers/1wire/1wire.c

```diff
@@ -45,6 +45,7 @@
 
 #include <nuttx/kmalloc.h>
 #include <nuttx/i2c/i2c_master.h>
+#include <nuttx/1wire/1wire_master.h>
 #include <nuttx/drivers/1wire.h>
```

Comment:
should we move 1wire.h from nuttx/drivers/ to nuttx/1wire too?

Reply:
Yes, you are right. I see also no reason why this is placed in the driver directory.